reference_docs/YAML_models/standoc_document_attributes.yaml

```diff
@@ -0,0 +1,724 @@
+    declaration: ":country:, :country{_i}:"
+    added_in_version: v1.9.4
+
+  postcode:
+    tags: ["Author info"]
+    title: "postcode"
+    description: "Postcode component of the organization address of a person who is a contributor to the document".
+    declaration: ":postcode:, :postcode{_i}:"
+    added_in_version: v1.9.4
+
+  contributor-uri:
+    tags: ["Author info"]
+    title: "Contributor URI"
+    description: "URI of a person who is a contributor to the document.".
+    declaration: ":pcontributor-uri:, :contributor-uri{_i}:"
+
+
+  email:
+    tags: ["Author info"]
+    title: "email"
+    description: "Email of a person who is a contributor to the document.".
+    declaration: ":email:, :email{_i}:"
+
+  phone:
+    tags: ["Author info"]
+    title: "phone"
+    description: "Phone number of a person who is a contributor to the document.".
+    declaration: ":phone:, :phone{_i}:"
+
+  fax:
+    tags: ["Author info"]
+    title: "fax"
+    description: "Fax number of a person who is a contributor to the document.".
+    declaration: ":fax:, :fax{_i}:"
+
+  subdivision:
+    tags: ["Author info"]
+    title: "Subdivision of the organization"
+    description: "Subdivision of the organization that is responsible for this document".
+    added_in_version: v1.6.1
+
+  subdivision-abbr:
+    tags: ["Author info"]
+    title: "Subdivision abbreviation of the organization"
+    description: "Abbreviation of the subdivision of the organization that is responsible for this document".
+    added_in_version: v1.6.1
+
+  pub-address:
+    tags: ["Author info"]
+    title: "Address of the organization"
+    description: |
+      "Address of the organization responsible for this document, if it overrides
+      the default.
+      +
+      [NOTE]
+      --
+      Each line in a multi-line address must end with `+ \`, e.g.
+
+      [source,adoc]
+      ----
+      :pub-address: 1 Infinity Loop + \
+      California + \
+      United States of America
+      ----
+      --".
+    added_in_version: v1.6.1
+
+  pub-phone:
+    tags: ["Author info"]
+    title: "Phone number of the organization"
+    description: "Phone number of the organization responsible for this document, if it overrides the default".
+    added_in_version: v1.6.1
+
+  pub-fax:
+    tags: ["Author info"]
+    title: "Fax number of the organization"
+    description: "Fax number of the organization responsible for this document, if it overrides the default".
+    added_in_version: v1.6.1
+
+  pub-email:
+    tags: ["Author info"]
+    title: "Email of the organization"
+    description: "Email of the organization responsible for this document, if it overrides the default".
+    added_in_version: v1.6.1
+
+  pub-uri:
+    tags: ["Author info"]
+    title: "URI of the organization"
+    description: "URI of the organization responsible for this document, if it overrides the default".
+    added_in_version: v1.6.1
+
+
+  #--- Visual appearance ---#
```

Comment:
No, and yes. I know that's not the answer you would like to see.

Best practice is (a) get familiar with the default CSS, so you know what you're overriding; https://www.metanorma.org/builder/topics/styling-output-html/ has not nearly enough information on CSS; (b) do not introduce arbitrary changes to any SDO-approved rendering; (c) any changes to look and feel should reflect a well-defined class of documents. (https://github.com/metanorma/annotated-express/issues/78 is where this came from: "ISO/TC 184/SC 4 has requested us to provide non-colored HTML output, i.e. the yellow/blue colors should be made grayscale." We made up the colours, but we were asked to do grayscale by one technical committee, not ISO, so that override applies to their class of documents.)

Risks: not following SDO branding and look-and-feel requirements; getting the CSS wrong; falling out of sync with any modifications we may do to the CSS (and we will, though we do try to avoid it)